examples/specs/histogram_nonlinear.vl.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
+  "description": "A histogram for a data source that provides non-linear bins. Thanks to [@Saba9](https://github.com/Saba9) who helps create this example.",
+  "data": {
+    "values": [
+      {
+        "startTime": "0",
+        "endTime": "8.33",
+        "residency": 0
+      },
+      {
+        "startTime": "8.33",
+        "endTime": "12.50",
+        "residency": 0
+      },
+      {
+        "startTime": "12.50",
+        "endTime": "16.67",
+        "residency": 31.17
```

Comment:
If you convert this to actual ratios, we can use the format `.2f%` below.

Reply:
what do you mean?